relay-dynamic-config/src/defaults.rs

```diff
@@ -176,6 +176,9 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                 Tag::with_key("span.main_thread")
                     .from_field("span.sentry_tags.main_thread")
                     .when(is_mobile.clone()),
+                Tag::with_key("span.app_start_type")
+                    .from_field("span.sentry_tags.app_start_type")
+                    .when(is_mobile.clone()),
```

Comment:
Do we need this tag on both `exclusive_time` and `exclusive_time_light` (i.e. both when we query with and without `transaction`)? Do we need it on any other metrics (e.g. `duration`)?

Reply:
`exclusive_time_light` is for when you query span metrics without the `transaction` tag.

Reply:
Thanks! Then I'll only store it for `exclusive_time`, this tag is only used with the `transaction` tag for now